### ISSUE TITLE
Invalid label value

### ISSUE THREAD

**Issue by RobiD** [cloudfoundry-community/firehose_exporter#16]
Every couple of days, firehose exporter stops responding due to a weird error, see below:
```
invalid label value "\x00\x00\x00\x01\xa8\n\xa5\x03\n\x03re"
```
This started happening after the last PCF upgrade. I assume the Pivotal guys did some changes in the way metrics are formatted within the Loggregator System. Or is this due to something else?

The current version of PCF is v1.10.6.0

Thanks!

**Comment by frodenas** [comment]
@RobiD Interesting error, I've tested the exporter on multiple PCF versions (1.9.x and 1.10.x) and have never seen this. Coming from the labels I can only think of an invalid host, but it's hard to say based on the error message. Is that all you get from the exporter logs?

**Comment by RobiD** [comment]
Dear @frodenas, we started getting a similar error now:
```
invalid metric name "firehose_exporter_value_metric_p_rabbitmq_p_rabbitmq_rabbitmq_queues_3681_af_6_a_26_c_5_44_ca_9_dc_3_5047_f_4_d_2582_f_granular_expected_loss_rating_exchange_anonymous_wwf_js_cpr_si_n_r_-_me_813_a_consumers"
```
It is constantly causing Prometheus to stop scraping metrics from PCF. The exporter itself seems to be working fine. Any suggestions?

**Comment by frodenas** [comment]
Thanks for reporting this @RobiD!

It should be fixed at https://github.com/cloudfoundry-community/firehose_exporter/commit/ec9e1d15b961d3182f9720f76665cb680fb6dbe1 (included at [v4.2.1](https://github.com/cloudfoundry-community/firehose_exporter/releases/tag/v4.2.1)). I'll cut a new BOSH release later today.

**Comment by RobiD** [comment]
Thanks a lot @frodenas, we have upgraded the Firehose Exporter and it works like a charm now! 😄  👍
